import-scripts/standardize_cna_data.py

```diff
@@ -0,0 +1,45 @@
+#! /usr/bin/env python
+
+""" standardize_cna_data.py
+This script rewrites a CNA file to replace all blank values with 'NA'.
+
+Usage:
+    python standardize_cna_data.py --filename $INPUT_CNA_FILE
+Example:
+    python standardize_cna_data.py --filename /path/to/data_CNA.txt
```

Comment:
our standard filename pattern is now 'data_cna.txt' for the cna file. I know that we use data_CNA.txt for the (legacy) dmp studies, but perhaps we will move those to the standard at some point. So I would recommend we use the standard filename 'data_cna.txt' in this file. It also makes me wonder if we should standardize our filenames in the delivered study but that's out of scope here.